content/docs/reference/addon-api/addon.tab/_index.md

```diff
@@ -348,6 +348,184 @@ Internally uses `window.django.gettext` or `window._messages`.
 
 See [addon.tab.appendToSharedSpace](addon.tab.appendtosharedspace).
 
+### `addon.tab.createModal`
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Type</th>
+    <th>Required</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>title</td>
+    <td><code>string</code></td>
+    <td>Yes</td>
+    <td>The title of the modal.</td>
+  </tr>
+  <tr>
+    <td>options</td>
+    <td><code>object</code></td>
+    <td>No</td>
+    <td>
+      <table>
+        <tr>
+          <th>Parameter</th>
+          <th>Type</th>
+          <th>Default</th>
+          <th>Description</th>
+        </tr>
+        <tr>
+          <td>isOpen</td>
+          <td><code>Boolean</code></td>
+          <td>false</td>
+          <td>Whether to open the modal by default.</td>
+        </tr>
+        <tr>
+          <td>useEditorClasses</td>
+          <td><code>Boolean</code></td>
+          <td>false</td>
+          <td>If in the editor, whether to apply the editor styles instead of the <code>scratch-www</code> ones.</td>
+        </tr>
+        <tr>
+          <td>useSizesClass</td>
+          <td><code>Boolean</code></td>
+          <td>false</td>
+          <td>If on <code>scratch-www</code>, whether to add the <code>modal-sizes</code> class.</td>
+        </tr>
+      </table>
+    </td>
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <td>Return value</td>
+    <td><code>Object</code></td>
+  </tr>
+</table>
+
+Returns a blank modal using Scratch's styles.

```

Comment:
The properties of the returned object (`content`, `open`, `close`, ...) should be listed somewhere.